controllers/limitador_controller.go

```diff
@@ -116,9 +118,39 @@ func (r *LimitadorReconciler) reconcileSpec(ctx context.Context, limitadorObj *l
 		return ctrl.Result{}, err
 	}
 
+	if err := r.reconcilePdb(ctx, limitadorObj); err != nil {
+		return ctrl.Result{}, err
+	}
+
 	return ctrl.Result{}, nil
 }
 
+func (r *LimitadorReconciler) reconcilePdb(ctx context.Context, limitadorObj *limitadorv1alpha1.Limitador) error {
+	logger, err := logr.FromContext(ctx)
+	if err != nil {
+		return err
+	}
+	if limitadorObj.Spec.PodDisruptionBudget == nil {
+		return nil
+	}
```

Comment:
If the PDB was initially  configured Limitador CR and  then removed from the spec, the PDB CR is still operationally  but not directly configured by limitador.  I would expect we would want to remove the PDB CR if the spec there is no spec configured in the Limitador CR.

Reply:
Good point, that's very valid. I'll add a commit to fix this

Reply:
Newest commit ad29718 added to handle this